ddtrace/payload.py

```diff
@@ -0,0 +1,84 @@
+import logging
+
+from .encoding import get_encoder
+
+log = logging.getLogger(__name__)
+
+
+class Payload:
+    __slots__ = ['traces', 'size', 'encoder', 'max_payload_size', 'filters']
+
+    # Default max payload size of 5mb
+    # DEV: Trace agent limit is 10mb, cutoff at 5mb to ensure we don't hit 10mb
+    DEFAULT_MAX_PAYLOAD_SIZE = 5 * 1000000
+
+    def __init__(self, encoder=None, filters=None, max_payload_size=DEFAULT_MAX_PAYLOAD_SIZE):
+        self.filters = filters
```

Comment:
We could have
```suggestion
        self.filters = filters or []
```

then we wouldn't need to `None` check `filters`

Reply:
I am going to keep as-is, because I would keep the `if not self.filters:` check anyways to check for an empty list.